pkg/gateway/multiparts/tracker_test.go

```diff
@@ -32,11 +32,6 @@ func TestTracker_Get(t *testing.T) {
 	testTrackerGet(t, tracker)
 }
 
```

Comment:
Just to make sure I understood your intention correctly - DB benchmarks are left intentionally?